Formula/ledger.rb

```diff
@@ -53,6 +53,7 @@ def install
     pkgshare.install "contrib"
     pkgshare.install "python/demo.py" if build.with? "python"
     elisp.install Dir["lisp/*.el", "lisp/*.elc"]
+    bash_completion.install pkgshare/"contrib/ledger-completion.bash"
```

Comment:
Can you please report to upstream that these files really should be handled by `make install` itself?

Reply:
Their bug tracker has been down for weeks http://bugs.ledger-cli.org/ and there's almost nothing with support for installing completions via `make install`, which is why I've approved this as-is.